content/learning-paths/servers-and-cloud-computing/cca-container/hello-realm.md

```diff
@@ -0,0 +1,132 @@
+---
+# User change
+title: "Run an application in a Realm"
+
+weight: 3 # 1 is first, 2 is second, etc.
+
+# Do not modify these elements
+layout: "learningpathall"
+---
+
+## Before you begin
+
+Make sure you have downloaded the [`armswdev/aemfvp-cca-image`](https://hub.docker.com/r/armswdev/aemfvp-cca-image) docker container.
+
+Install [GCC](/install-guides/gcc/) on your machine. Dependent on the architecture of your machine, you will need to install either the native compiler or cross-compiler.
+
+## Overview
+
+To run an application inside a Realm, you will package it as the "init" process for the guest linux kernel. 
+
+Linux kernels contain a gzipped cpio format archive. When the kernel boots up, this archive is extracted into the root filesystem. The kernel then checks whether the root filesystem contains a file named "init" and tries to run it. At this point the kernel is done booting, and the "init" application executes the system the rest of the way. 
+
+In this section, you will package a simple hello world application into the initramfs for the guest linux kernel. 
+
```

Comment:
Might be worth setting some clear expectations about how this will work. The use of the `init` process to run the app means that the experiment will become non-interactive. There will not be a shell prompt in the guest as per the previous step. The guest VM will simply run the app and then exit.

Reply:
Updated with clear expectations around usage.